### PR TITLE
Fix/write tests

### PR DESCRIPTION
**Pull request by jmettler27**
This fix adds missing unit and instrumented tests

### CODE REVIEW

**Comment by Sou-ly** [comment]
What is "life-cycle"? I don't understand what it's supposed to be doing.

**Comment by jmettler27** [comment]
> What is "life-cycle"? I don't understand what it's supposed to be doing.
Where do you see it?

**Review by NicolasMatekalo — APPROVED** [review]

File: app/src/test/java/com/github/sdp/ratemyepfl/model/review/ReviewTest.kt [hunk 1]
```diff
@@ -81,7 +98,7 @@ class ReviewTest {
 
     @Test
     fun builderSetTitleCorrectly() {
-        val title: String = "My title"
+        val title = "My title"
```
Comment (emilio-gambino) [inline app/src/test/java/com/github/sdp/ratemyepfl/model/review/ReviewTest.kt:101]: Nice catch !

**Review by emilio-gambino — APPROVED** [review]
Thanks for this work, we really needed more coverage !

**Pull request merged**